core/migrations/10_deploy_designated_voting_factory.js

```diff
@@ -5,5 +5,7 @@ const { getKeysForNetwork, deploy } = require("../../common/MigrationUtils.js");
 module.exports = async function(deployer, network, accounts) {
   const keys = getKeysForNetwork(network, accounts);
 
-  await deploy(deployer, network, DesignatedVotingFactory, Finder.address, { from: keys.deployer });
+  const finder = await Finder.deployed();
```

Comment:
Any specific reason for this refactor?

Reply:
I don't think buidler allows you to do `Finder.address`. You have to call `let finder = await Finder.deployed()` and then call `finder.address` on the returned instance.

Reply:
Yeap, thats precisely it